pkg/apis/azureprovider/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,7 @@
 // +build !ignore_autogenerated
```

Comment:
This is the culrprit. We need to account for special cases like this. AWS is doing the same, so we should follow suit IMO

Reply:
It's only part of the problem though.
- Their script differs from the one in k/k
- We're using different tools for testing
- I think they're using different tags/tools for generating code

Until I get a chance to investigate a little of that, I'm going to hold off on editing the script.